mix.exs

```diff
@@ -41,7 +42,33 @@ defmodule Tokenizers.MixProject do
       main: "Tokenizers",
       source_ref: "v#{@version}",
       source_url: @source_url,
-      extras: ["LICENSE", "notebooks/pretrained.livemd"]
+      extras: ["LICENSE", "notebooks/quicktour.livemd", "notebooks/pretrained.livemd"],
+      groups_for_docs: [
+        Creation: &(&1[:section] == :creators),
+        Setup: &(&1[:section] == :setup),
+        Inference: &(&1[:section] == :infer),
+        Training: &(&1[:section] == :train)
+      ],
```

Comment:
I like to use those only when we have really large models. For smaller modules, it actually makes things harder, because you need to open up multiple subcategories to see all functions. I think none of our modules are currently big enough to justify it. Let's remove this entry and also the `@doc section: ...` tags?

Reply:
I've created it only for `tokenizer` because it seemd to be large enough and good fits for structuring.